### PR TITLE
Feature/issue 77/bird view update

### LINKED ISSUE #77: Upgrading birdview for focus purposes
Trying to add a clickable bird view for faster navigation through bigger graphs.

### PR DESCRIPTION
**Pull request by trami25**
closes #77  Birdview is now interacting with the main view and tree view where the node will be selected on all three views when clicked on it

### CODE REVIEW

File: block_visualizer_plugin/src/template/block_visualizer_template.jinja2 [hunk 1]
```diff
@@ -437,6 +435,8 @@
             d.fy = null;
 	    updateBirdView();
         }
+        birdView.on("click", function (event) {
+});
```
Comment (vradovic) [inline block_visualizer_plugin/src/template/block_visualizer_template.jinja2:439]: This seems to be an empty function :thinking:

File: simple_visualizer_plugin/src/template/simple_visualizer_plugin/simple_visualizer_template.jinja2 [hunk 2]
```diff
@@ -196,6 +196,8 @@
                 correspondingNode.classed("focused", true);
             }
 
+        birdView.on("click", function (event) {
+        });
```
Comment (vradovic) [inline simple_visualizer_plugin/src/template/simple_visualizer_plugin/simple_visualizer_template.jinja2:200]: This also seems to be empty.

File: block_visualizer_plugin/src/template/block_visualizer_template.jinja2 (hunk 1 above)
Comment (trami25) [reply block_visualizer_plugin/src/template/block_visualizer_template.jinja2:439]: It allows bird view to be sensitive on click, but it doesn't specifically do until I click on nodes whose method is separated. If I don't have it, the view, won't recognize the click

**Review by vradovic — APPROVED** [review]
Looks good.

**Pull request merged**